README.md

```diff
@@ -106,6 +106,34 @@ item.created? # => true
 item.selected? # => false
 ```
 
+### Filter methods
+
+Plugin can define dataset methods for all enum values:
+
+```ruby
+Item.plugin :enum_values, filter_methods: true # default is `false`
+
+Item.dataset.type_first # => #<Sequel::Dataset: "SELECT * FROM \"items\" WHERE (\"type\" = 'first')">
+Item.dataset.type_second # => #<Sequel::Dataset: "SELECT * FROM \"items\" WHERE (\"type\" = 'second')">
+
+Item.dataset.status_created # => #<Sequel::Dataset: "SELECT * FROM \"items\" WHERE (\"status\" = 'created')">
+Item.dataset.status_selected # => #<Sequel::Dataset: "SELECT * FROM \"items\" WHERE (\"status\" = 'selected')">
```

Comment:
I suggest to define methods like `.selected`. `.first` `type` will be bad for this, but we can redefine values of `type`, it's anyway non-real, like `one, another, yet_another`.

Reply:
Done.